compiler/options.nim

```diff
@@ -40,7 +40,8 @@ type                          # please make sure we have under 32 options
     optMemTracker,
     optHotCodeReloading,
     optLaxStrings,
-    optNilSeqs
+    optNilSeqs,
+    optOldAst
```

Comment:
Maybe this should be a ``TGlobalOption`` instead? Not sure though.

Reply:
I didn't really know either, I just copied what nilseqs did.  I suppose you could set it in some modules and not others, and it should still work ok.

Reply:
Ok.